requirements.txt

```diff
@@ -1,3 +1,3 @@
 numpy >= 1.16.6
-onnx >= 1.2.3
```

Comment:
@pranavsharma , @snnn , @faxu and I had a discussion and we decided we want to remove the onnx requirement entirely (not just for aarch64)
the onnxruntime python api has no dependency on onnx python package.
it's only needed for onnx backend api, which we will support 'unofficially'

Reply:
Makes sense.

Reply:
Can you also please add a note in backend.py that this is not the official Python API? We'd also discussed removing the mention of the backend API here https://microsoft.github.io/onnxruntime/python/api_summary.html#backend. Can be in a different PR.

Reply:
Added the note.